core-aam/core-aam.html

```diff
@@ -734,10 +734,10 @@
                         <td>
 				<code>ROLE_SYSTEM_GROUPING</code> + object attribute <code>xml-roles:feed</code></td>
                         <td>
-                            Need to verify. 
-			    <ul><li>Control Type: <code>Group</code></li>
-                                <li>Localized Control Type: <code>"feed"</code></li>
-                            </ul>
+			                    <ul>
+                            <li>Control Type = <code>Group</code></li>
+                            <li>Localized Control Type = "<code>feed</code>"</li>
```

Comment:
I noted in [PR 468](https://github.com/w3c/aria/pull/468#discussion_r89543308) that the current sytle was to use "is" not "=".  It appears the document inconsistently uses "is" in some cases and ":" (colons) in others.  Personally, I prefer the colons, but you are the UIA expert.  What would be best here, @boggydigital?  I will make a global change based on your advice.